src/lib/crypto/kimchi_backend/common/plonk_constraint_system.ml

```diff
@@ -144,6 +172,8 @@ module Plonk_constraint = struct
       | EC_endoscale of
           { state : 'v Endoscale_round.t array; xs : 'v; ys : 'v; n_acc : 'v }
       | EC_endoscalar of { state : 'v Endoscale_scalar_round.t array }
+      | Raw of
```

Comment:
Oh wow this is very interesting! I assume this will be for single-row gates and if we need to build one plonk constraint type then we can call the Raw with the corresponding parameters?

Reply:
This is just supposed to be for my use while testing, but in theory yes!